src/utils/layout.sh

```diff
@@ -18,3 +18,10 @@ auto_balance() {
   # Balance the tree rooted at some node automatically.
   bspc node "$1" -B
 }
+
+# calc to rounded integer
+rcalc() {
+  expr=${@}
+
+  echo "scale=1; $expr" | bc | cut -d. -f1
```

Comment:
Note that setting `scale` to `1` here allows folks to use fractional ratio representations like `1 / 3` (albeit with noticeably imperfect accuracy for irrational values).